jsapp/xlform/src/model.surveyDetail.coffee

```diff
@@ -6,16 +6,26 @@ module.exports = do ->
   #     start time, deviceid, (etc.)
   class SurveyDetail extends base.BaseModel
     idAttribute: "name"
+
     toJSON: ()->
       if @get("value")
-        nameSlashType = @get("name")
-        name: nameSlashType
-        type: nameSlashType
+        out = {}
+
+        out.name = @get("name")
+        # type is same as name
+        out.type = out.name
```

Comment:
Does it make sense to you that this always uses the name as the type? In https://github.com/kobotoolbox/kpi/blob/2187-audit/jsapp/xlform/src/model.configs.coffee, there's a `type` inside the `asJson` object for each setting, but I can't tell where that's used for anything except this one check: https://github.com/kobotoolbox/kpi/blob/f62a0c58a0b4c90c6e5a96182feb86a54a74dcfb/jsapp/xlform/src/model.surveyFragment.coffee#L346-L356

Of course, all the `name`s in the `asJson`s happen to be identical to the `type`s :slightly_smiling_face:

Reply:
https://github.com/kobotoolbox/kpi/issues/2467